dex/src/test/scala/com/wavesplatform/dex/settings/BaseSettingsSpecification.scala

```diff
@@ -155,6 +155,12 @@ class BaseSettingsSpecification extends AnyFlatSpec {
          |          client.bar = 3
          |        }
          |      }
+         |
+         |      circuit-breaker {
+         |        max-failures = 999
+         |        call-timeout = 123s
+         |        reset-timeout = 1d
```

Comment:
Do you have some tests for circuit-breaker? 
I can suggest unit test in `MatcherApiRouteSpecification`

Reply:
The circuit breaker we are using is implemented and well [tested](https://github.com/akka/akka/blob/master/akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala) in Akka.

Will add an integration test that checks we can continue work when Kafka issues are solved.